packages/flame_lottie/lib/src/lottie_component.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:async';
+
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame_lottie/src/lottie_renderer.dart';
+import 'package:flutter/widgets.dart';
+import 'package:lottie/lottie.dart';
+
+/// A flame [Component] which renders a [Lottie] animation using
+/// the already existing Flutter library [lottie](https://pub.dev/packages/lottie)
+///
+/// @param controller The controller which drives the animation. In case none is
```

Comment:
`@param` is not recommended to use:
https://dart.dev/guides/language/effective-dart/documentation#do-use-prose-to-explain-parameters-return-values-and-exceptions